ffe.py

```diff
@@ -11,7 +12,9 @@ def __init__(self):
         self.settings = Settings()
         # Load the font, a 32 by 8 tile font with libtcod's old character layout.
         self.tileset = tcod.tileset.load_tilesheet(
-            "resources\\images\\dejavu10x10_gs_tc.png", 32, 8, tcod.tileset.CHARMAP_TCOD,
+            os.path.abspath("resources/images/dejavu10x10_gs_tc.png"),
```

Comment:
I would recommend joining the current path instead of returning an absolute path. Absolute paths work differently on Unix and Windows machines. 

Try  os.path.join(os.path.dirname(__file__), 'resources/images/dejavu10x10_gs_tc.png')

Reply:
They do. However, it will give you an absolute path for Linux if you are on a Linux machine or windows if you're on a windows machine. This also gets rid of the issue where we have to worry about setting up a working folder in a virtual environment. I.e. one does not need to change anything if they are not in the current working folder. os.path.abspath will work for any operating system.

Reply:
Does there need to be an error check with os.path.lexists()?

Reply:
For sure, we can make an issue for it to be put in a try block.